src/main/java/com/minecolonies/commands/RespawnCitizen.java

```diff
@@ -0,0 +1,120 @@
+package com.minecolonies.commands;
+
+import com.blockout.Log;
+import com.minecolonies.colony.CitizenData;
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.entity.EntityCitizen;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * List all colonies.
+ */
+public class RespawnCitizen extends AbstractSingleCommand
+{
+
+    private static final String CITIZEN_DESCRIPTION = "§2ID: §f %d §2 Name: §f %s";
+    private static final String REMOVED_MESSAGE                 = "Has been removed";
+    private static final String NO_COLONY_CITIZEN_FOUND_MESSAGE = "No citizen %d found in colony %d.";
+    private static final String COORDINATES_XYZ                 = "§4x=§f%s §4y=§f%s §4z=§f%s";
+    private static final String CITIZEN_DATA_NULL               = "Couldn't find citizen client side representation of %d in %d";
+    private static final String ENTITY_CITIZEN_NULL             = "Couldn't find entity of %d in %d";
+    private static final String COLONY_NULL                     = "Couldn't find colony %d";
+
+    public static final String DESC                             = "respawn";
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public RespawnCitizen(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @NotNull
+    @Override
+    public String getCommandUsage(@NotNull final ICommandSender sender)
+    {
+        return super.getCommandUsage(sender) + "<ColonyId> <CitizenId>";
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        final int colonyId = getIthArgument(args, 0, -1);
+        final int citizenId = getIthArgument(args, 1, -1);
+
+        //todo add this in a feature update when we added argument parsing and permission handling.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)